packages/genesys-spark-components/src/components/stable/gux-icon/gux-icon.tsx

```diff
@@ -100,6 +100,13 @@ export class GuxIcon {
       svgElement.setAttribute('aria-hidden', String(this.decorative));
     } else {
       svgElement.setAttribute('aria-hidden', 'false');
+
+      /*
+        If the icon is not decorative, we need to set the role to img to ensure that
+        the screenreader will announce the localized screenreader-text property.
+        https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role#svg_and_roleimg
+      */
+      svgElement.setAttribute('role', 'img');
```

Comment:
NVDA was reading the icons with screenreader text as blank so this change fixes that.

Reply:
In Firefox, it's down as a 'graphics-document'

Reply:
@daragh-king-genesys happy to move this part to a separate issue in the backlog if you'd prefer

Reply:
Ya move it into a seperate issue